### ISSUE TITLE
[Request] Reload retroarch after exiting Dolphin

### ISSUE THREAD

**Issue by SwedishGojira** [RobLoach/libretro-dolphin-launcher#11]
If you could add an option to relaunch RetroArch after exiting Dolphin instead of just quitting that would be awesome.

**Comment by RobLoach** [comment]
I don't think it quits RetroArch afterwards. How are you quitting Dolphin? Maybe the Escape key repeats and quits both? Maybe the core is crashing?

It actually just runs Dolphin on top of RetroArch and doesn't quit while running.

**Comment by i30817** [comment]
Wasn't there a setting in RA to quit the gui while a core is running to save CPU? Might be imagining it.
